docs/ref/contrib/gis/install/geolibs.txt

```diff
@@ -19,6 +19,10 @@ Program                   Description                           Required
 Note that older or more recent versions of these libraries *may* also work
 totally fine with GeoDjango. Your mileage may vary.
 
+.. versionchanged:: 1.11
+
+    GDAL is now required independently from the database backend.
```

Comment:
In older versions, GDAL is required only for SQLite. Now it's required for all databases.